src/app/code/community/Shopgate/Cloudapi/Model/Api2/Customers/Rest/Admin/V2.php

```diff
@@ -45,4 +45,71 @@ protected function _update(array $filteredData)
     {
         $this->_critical(self::RESOURCE_METHOD_NOT_ALLOWED, Mage_Api2_Model_Server::HTTP_FORBIDDEN);
     }
+
+    /** @noinspection PhpHierarchyChecksInspection */
+    /**
+     * @param array $filteredData
+     *
+     * @return array
+     * @throws Mage_Core_Exception
+     */
+    protected function _create($filteredData)
+    {
+        /** @var Mage_Customer_Model_Customer $customer */
+        $customer = Mage::getModel("customer/customer");
+
+        $newCustomerData = new Varien_Object();
+        $newCustomerData->setData($filteredData);
+        $newCustomerData->setStore($this->getStore($newCustomerData));
+        $newCustomerData->setWebsiteId($this->getWebsiteId($newCustomerData));
+
+        $customer->setData($newCustomerData->getData());
+
+        try {
+            $customer->save();
+        } catch (Exception $e) {
+            $this->_critical($e->getMessage());
+        }
+
+        return $this->filterOutData($customer->getData());
+    }
+
+    /**
+     * @param Varien_Object $requestData
+     *
+     * @return Mage_Core_Model_Store
+     * @throws Mage_Api2_Exception
+     */
+    protected function getStore($requestData)
```

Comment:
just noticed something, store_id is not even a required parameter to pass. The only thing the customer needs is a website_id. See https://devdocs.magento.com/guides/m1x/api/rest/Resources/resource_customers.html#RESTAPI-Resource-Customers-HTTPMethod-POST-customers